### DISCUSSION TITLE
Cosmos DB gremlin API

### DISCUSSION THREAD

**Discussion by martijn-bro** [aws/graph-explorer#534] [Q&A]
Hi all,
just a general question. Has anyone made this tool work by deploying as a webapp and connecting it to a cosmos db gremlin database?

**Comment by kmcginnes** [comment]
@martijn-bro I haven't tried that explicitly. As long as it has HTTP Gremlin endpoints that can be queried it should work.

Let us know if you have issues.

**Reply by martijn-bro** [reply]
Cheers, i'll have a play around with it :)
